donation-tracker/src/components/campaignCard.tsx

```diff
@@ -18,6 +19,11 @@ interface Props {
 const CampaignCard = ({ campaign, donationItems, big }: Props) => {
   const { navigate } = useI18next();
   const { t } = useTranslation();
+  const langContext = React.useContext(I18nextContext);
+
+  const title = getCampaignTitle(campaign, langContext.language);
```

Comment:
just for clarification: `langContext.language` will always be set to one of the supported languages, right? Including things like fallback to the preconfigured default language?

Reply:
yes, it always contains the detected one - or the fallback